src/pkg/errs/core/core.go

```diff
@@ -61,6 +61,10 @@ var (
 	ErrBackupNotFound = &Err{msg: "backup not found"}
 	// a catch-all for downstream api auth issues.  doesn't matter which api.
 	ErrInsufficientAuthorization = &Err{msg: "insufficient authorization"}
+	// basically what it sounds like: we went looking for something by ID and
+	// it wasn't found.  This might be because it was deleted in flight, or
+	// because we had the id wrong, or some other reason.
+	ErrNotFound = &Err{msg: "not found"}
```

Comment:
Nit:
The changes look good, but I'm not sure sure about the part of the comment "... because we had the id wrong ...".
How would we end up with a wrong id? And even if we do, should it be considered same as "deleted in flight"?

Reply:
> How would we end up with a wrong id?

Well, hopefully we never would, right?  Heh, I'm probably adding more confusion than I'm alleviating here.  I'll remove that part.  Thanks for pointing it out.